DockerfileNEW

```diff
@@ -0,0 +1,80 @@
+# Stage: base image
+FROM node:18.12-bullseye-slim as base
+
+ARG BUILD_NUMBER=1_0_0
+ARG GIT_REF=not-available
+
+LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
+
+ENV TZ=Europe/London
+RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
+
+RUN addgroup --gid 2000 --system appgroup && \
+        adduser --uid 2000 --system appuser --gid 2000
+
+WORKDIR /app
+
+# Cache breaking
+ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
+
+RUN apt-get update && \
+        apt-get upgrade -y 
+        # apt-get autoremove -y && \
+        # rm -rf /var/lib/apt/lists/*
+
+RUN apt-get install -y curl
+
+RUN curl https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem > /app/root.cert
+
+# Stage: build assets
+FROM base as build
+
+ARG BUILD_NUMBER=1_0_0
+ARG GIT_REF=not-available
+
+RUN npm install -g npm@9
+
+RUN apt-get install -y make python3 g++
+
+COPY package*.json ./
+RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit
```

Comment:
Gradle tests used for Integration tests, so I guess the cypress reference can be removed?